enes-homework-week1/app.js

```diff
@@ -0,0 +1,163 @@
+    listItemMaker();
+    bookListMaker();
+    bookCover();
+  }
+
+  function listItemMaker() {
+    for (let i = 0; i < listOfBooks.length; i++) {
+      const li = document.createElement('li');
+      li.innerText = listOfBooks[i];
+      root.appendChild(li);
+    }
+  }
+
+  //1.4
+
+  const objListOfBooks = {
+    ak: {
+      title: 'book1',
+      author: 'author1',
+      language: 'language1',
+    },
+    bak: {
+      title: 'book2',
+      author: 'author2',
+      language: 'language2',
+    },
+    cak: {
+      title: 'book3',
+      author: 'author3',
+      language: 'language3',
+    },
+    dak: {
+      title: 'book4',
+      author: 'author4',
+      language: 'language4',
+    },
+    eak: {
+      title: 'book5',
+      author: 'author5',
+      language: 'language5',
+    },
+    fak: {
+      title: 'book6',
+      author: 'author6',
+      language: 'language6',
+    },
+    gak: {
+      title: 'book7',
+      author: 'author7',
+      language: 'language7',
+    },
+    hak: {
+      title: 'book8',
+      author: 'author8',
+      language: 'language8',
+    },
+    iak: {
+      title: 'book9',
+      author: 'author9',
+      language: 'language9',
+    },
+    jak: {
+      title: 'book10',
+      author: 'author10',
+      language: 'language10',
+    },
+  };
+
+  //1.5 1.6
+  function bookListMaker() {
+    const div = document.createElement('div');
+    root.appendChild(div);
+    const h2 = document.createElement('h2');
+    h2.innerText = 'My Second Book Info List';
+    div.appendChild(h2);
+    const div1 = document.createElement('div');
+    div.appendChild(div1);
+    div1.className = 'container';
+    for (const key in objListOfBooks) {
+      const div2 = document.createElement('div');
+      div2.className = 'wrapper';
+      const x = Math.floor(Math.random() * 256);
+      const y = Math.floor(Math.random() * 256);
+      const z = Math.floor(Math.random() * 256);
+      const bgColor = 'rgb(' + x + ',' + y + ',' + z + ')';
+      div2.style.background = bgColor;
+      div1.appendChild(div2);
+      const h2 = document.createElement('h2');
+      h2.innerText = objListOfBooks[key]['title'];
+      div2.appendChild(h2);
+      const p1 = document.createElement('p');
+      p1.innerText = 'author: ' + objListOfBooks[key]['author'];
+      div2.appendChild(p1);
+      const p2 = document.createElement('p');
+      p2.innerText = 'language: ' + objListOfBooks[key]['language'];
+      div2.appendChild(p2);
+    }
+  }
+
+  //1.7
+
+  const objBookCovers = {
+    aak: 'bookcovers/cover1.jpg',
+    bak: 'bookcovers/cover2.jpg',
+    cak: 'bookcovers/cover3.jpg',
+    dak: 'bookcovers/cover4.jpg',
+    eak: 'bookcovers/cover5.jpg',
+    fak: 'bookcovers/cover6.jpg',
+    gak: 'bookcovers/cover7.jpg',
+    hak: 'bookcovers/cover8.jpg',
+    iak: 'bookcovers/cover9.jpg',
+    jak: 'bookcovers/cover10.jpg',
+  };
+
+  //1.8
+
+  function bookCover() {
+    const arrOfKeys = Object.keys(objBookCovers);
```

Comment:
You defined the `arrOfKeys` but have not used it in the function. Later you are using the for .. in loop to go over `objBookCovers` again. 

Here is a nice discussion about the two different ways to loop over items https://www.reddit.com/r/javascript/comments/8emf94/forin_vs_objectkeys/ & https://zellwk.com/blog/looping-through-js-objects/

Some nice reading material 💯